### ISSUE TITLE
[define] Collective AI:

### ISSUE THREAD

**Issue by drrosa** [collai/community#1]
# Collective AI
A collaborative workspace for anyone who wants to learn, develop technical skills, and become a better practitioner in the field of Artificial Intelligence.

# <img width="250" alt="cropped_black" src="https://user-images.githubusercontent.com/2343554/46038923-442fbb00-c0c1-11e8-901f-49ef05f0d0f6.png">

**σ** :: A _self-directed_ learning workspace for AI projects
 –Allows everyone to _deviate_ from accepted standards, develop their skills, and become better AI practitioners.

**Σ** :: A _community-driven_ effort that is powered by the _sum_ of its individual members
–Brings more talent and unique perspectives to the field.

**ι** :: An _inclusive_ resource pipeline where members are encouraged to _invite_ others to join
–Connects those in the field with those who want to become part of the community at large.

**∆** :: A _diverse_ network of AI practitioners in constant _change_
–Supports growth according to the individual's goals and the community's best interests.

**Core Guidelines:**
**1.** Do and say things that add value to each individual and the community as a whole.

**Join Here! –> [https://collai.zulipchat.com](https://goo.gl/D6NvXi)**

**Comment by drrosa** [comment]
## Contents:
- [Introduction](#issue-363024480)
- [Too Long; Didn't Read](#issuecomment-425027967)
- [Powered by Zulip](#issuecomment-425028038)
- [Why Join?](#issuecomment-425028129)
- [Core Values](#issuecomment-425028276)
- [Core Guidelines](#issuecomment-425028668)
- [Community Framework](#issuecomment-425229425)

**Comment by drrosa** [comment]
## TL;DR
This community is for anyone who wants to devote time to the art and science of AI; regardless of their formal role or stage of life. It doesn’t matter whether you consider yourself a student, an artist, a hacker, a scholar, or something else entirely. Everyone is welcome no matter who they are or where they come from––as long as they are willing to work hard on their technical skills and become better practitioners of AI. If anything here resonated with you, please come join!

**Comment by drrosa** [comment]
## Powered by Zulip
We are going to be using open source software as our primary tool for community interaction. At a glance, one might think [Zulip](https://zulipchat.com/for/open-source) is yet another group chat software. However, anyone who starts digging into it eventually realizes it’s a powerful and [extensible](https://github.com/zulip) medium for communication.

We need software that can support a well-designed information architecture to maintain the community's ontological knowledge base. In other words, such software should have capabilities that can support a community effort to keep discussions as organized as possible now and in the forseeable future.

The contents of the dicussions that will take place in this community are only as valuable as they are discoverable. Zulip has many features that help maximize both content value and discoverability for its users, including, but not limited to the following:

- [Streams and topics](https://zulipchat.com/help/about-streams-and-topics): Think chat channels + email subjects. It combines real-time chat with threaded conversations. This combination makes it easier to sift through hundreds of messages, narrow down, find what's most relevant, and participate in multiple discussions
- Native support for [Markdown, LaTeX, and syntax highlighting](https://zulipchat.com/help/format-your-message-using-markdown) in the message editor
- Custom [auto-linkification](https://zulipchat.com/help/add-a-custom-linkification-filter) filtering using regular expressions e.g. #4242 gets auto-converted into a link to a PR or GitHub issue
- [Interactive chatbots](https://zulipchat.com/api/running-bots) deployable by any user
- A growing number of [custom integrations](https://zulipchat.com/integrations/)
- Auto-generation of link previews
- Drag and drop file uploads
- Custom Emojis
- It's open source!

**Comment by drrosa** [comment]
## Why Join?
There are plenty of great online resources to study AI. An increasing and perhaps even overwhelming number of them. Anyone can go through existing resources alone or together in whatever combination they want. However, the accountability that comes from participation within a group of peers is what separates spectators from doers.

**Comment by drrosa** [comment]
## Core Values
The goal is to design a self-sustaining platform that fosters self-directed and community-driven growth through inclusion and diversity.

**Sounds nice, but what does that even mean? And what’s with the greek letters (σ Σ ι ∆)?**

**“Self-directed”** refers to learning under one’s own control.
**σ (sigma)** alludes to the _standard deviation_ and is a reminder that we are free to _deviate_ as close to or as far away as we like from the average learning path of choice.

**“Community-driven”** refers to the understanding that in community we add to each other's’ lives by learning from and teaching one another.
**Σ (Sigma)** alludes to the _summation operator_ and is a reminder that a community is essentially the _vector sum_ of every single interaction among individual community members.

**“Inclusion”** refers to being accessible to anyone no matter who they are or where they come from.
**ι (iota)** alludes to the _inclusion function_ and is a reminder that members are encouraged to _include_ others via personal invitations and promote an all-inclusive community.

**“Diversity”** is really a quality that emerges as a natural consequence of bringing together a set of individuals who are self-directed, community-driven, and inclusive. However, it also refers to the celebration of uniqueness of backgrounds.
**∆ (Delta)** alludes to _variation_ and is a reminder that at the most fundamental level, diversity is change.

**“Growth”** is the final result. It also refers to the way in which one grows both as a person and in technical skills, according to one’s own interests and goals; while at the same time staying aligned with the community’s best interests.

These core values have been heavily influenced by the [Recurse ![rc_mini](https://user-images.githubusercontent.com/2343554/46165089-8e8d7500-c244-11e8-89d0-c81483515bdc.png) Center](https://www.recurse.com/scout/click?t=423bd116a0d56246ab76ef8e36234987). They have written a [great piece](https://www.recurse.com/blog/138-what-we-mean-by-self-directed-and-community-driven) on the subject. Their continued success relies on the fact that they have always stayed true to their values for over seven years and counting.

_Unfortunately, no amount of writing can fully convey the importance and the outcome of a community with such values at its core. The best way to understand the kind of environment produced by this type of community is to see and experience it for yourself._

**Comment by drrosa** [comment]
## Core Guidelines
There’s really only one thing we need to remember at all times:

**Do and say things that add value to each individual and the community as a whole.**

The following four habits go a long way toward creating value within any community, but especially when it comes to learning:

**Cool Habits:**
**1.** Use your mind.
**2.** Be authentic.
**3.** Be fully involved in a conversation.
**4.** Be respectful and tolerant.

They may sound obvious, but in reality, most people don’t realize they are in the habit of doing the opposite. So let’s break it down into the four most common ways in which people tend to lose their cool. Staying away from practicing the following habits really makes a tremendous difference when it comes to learning collaboratively. The idea is to strive to remove all barriers to learning together.


**Uncool Habits**
**1.** Starting a sentence with something along the lines of “Well, actually …"
This sort of interjections pretty much never adds significant value. To be blunt, this is usually done by someone wanting to show off their knowledge, those who like trolling other people, or someone who doesn’t realize they do it out of being “excellent at the skill of differentiation”–which inevitably leads to some degree of [pedantic habituation](http://basicbitch.software/posts/2017-07-11-The-Art-of-Differentiation.html).

It is a good idea to call someone out when such behavior is seen. But please do so via private message or in public using kindness/humor, if you feel skilled enough to do so. In general, it is usually best to express disapproval in private and praise in public. There are very few counterintuitive and rare exceptions where such an interjection would actually add value. Please use your mind. When in doubt, try asking yourself things like: “Would it be relevant to the conversation?”, “Does it need to be said right now?”, or “Would it add value to the conversation?”, and then act accordingly.

In this context, it’s irrelevant whether we are right or able to correct a tiny misconception. This guideline is all about creating an atmosphere that’s conducive to collaboration and maximizing the possibility of a positive learning experience for everyone involved.

**2.** Acting surprised or faking surprise when someone doesn’t know something you think everyone is supposed to know. If you think about it, there is really no need to do that. Be authentic. 
Instead of saying: “O-M-G, you don’t know what VIM is???”
Simply say: Vim is a code editor and you can learn more about it here https://www.vim.org.
Another good way to prevent such scenarios is to avoid using too many acronyms.

**3.** Interrupting a conversation that you are not going to stay completely engaged in. It is best to participate in an exchange of ideas by getting fully involved in the conversation that’s taking place.

**4.** Making subtle or explicit demeaning remarks about other people’s, gender, identity, race, background, etc. or being an asshole in general. The solution to this one is rather easy. Just don’t do it. Alternatively, learn how to be respectful and tolerant towards others. [Better late than never](https://www.newyorker.com/science/elements/after-years-of-abusive-e-mails-the-creator-of-linux-steps-aside).

_Fortunately, uncool habits are just habits that can be broken. But sometimes they are not easy to break. Let’s try our best to help one another. Please try to be forgiving and open-minded with those who violate our one community guideline._

> **Do and say things that add value to each individual and the community as a whole.**

**Comment by drrosa** [comment]
## Community Framework
* The following is just a sneak peek and is subject to change :)

### User Profile
![zulip_profile](https://user-images.githubusercontent.com/2343554/46510211-5665d380-c7fc-11e8-8a7e-b42cf291edeb.png)
## Communication Channels

#### Main Categories
The following naming conventions help keep streams categorized and visibly grouped together.

**/σ/** -> Self-directed learning resources for technical growth
**/Σ/** -> Community-driven activities for personal growth
**/ι/** -> Inclusion initiatives to increase and maintain an active user base
**/∆/** -> Diversity corner for non-technical, non-AI, non-etc i.e. A celebration of diversity in life

#### Sub-categories
The following provides more detail as to what type of content should go in a particular stream.

**a/** administrative
**b/** books or any reading material
**c/** complete courses
**h/** get help, feel heard
**p/** practical discussions, project collaboration
**r/** scholarly research
**s/** suggestions, sharing, discussions, socializing
**v/** videos, animations, movies

### Self-directed Learning Streams
![zulip_tech_streams](https://user-images.githubusercontent.com/2343554/46510210-5665d380-c7fc-11e8-90b7-2f6ad5001a43.png)
### Community-driven Streams
![zulilp_community_streams](https://user-images.githubusercontent.com/2343554/46510208-55cd3d00-c7fc-11e8-9267-10d050f26610.png)
### Inclusivity Streams
![zulip_inclusive_streams](https://user-images.githubusercontent.com/2343554/46510207-55cd3d00-c7fc-11e8-8ea0-28e49b23c9dd.png)
### Diversity Streams
![zulip_diverse_streams](https://user-images.githubusercontent.com/2343554/46510209-55cd3d00-c7fc-11e8-87e0-f5b310834562.png)

**Comment by drrosa** [comment]
I would like to invite everyone who is interested to leave a comment. However, I would like to request feedback from specific people first. If you receive a direct email or mention from me, please feel free to share your thoughts and comments here. Thanks!

**Comment by munashe5** [comment]
Great stuff! I support this idea. 
Is there any way you can add more details about Zulip? Maybe a few bullet points about what makes it great. The current description leaves me hanging a bit. 
Also, I recommend talking to women in the field. Tech field tends to have a terrible gender bias and so the sooner you begin to think about not only including women but having the platform be shaped by female voices the better. 
I hope that helps

**Comment by drrosa** [comment]
> Great stuff! I support this idea.
> Is there any way you can add more details about Zulip? Maybe a few bullet points about what makes it great. The current description leaves me hanging a bit.

@munashe5 You are right. I've added a [few bullet points](#issuecomment-425028038), and a [few screenshots](#issuecomment-425229425) too!

> Also, I recommend talking to women in the field. Tech field tends to have a terrible gender bias and so the sooner you begin to think about not only including women but having the platform be shaped by female voices the better.

I completely agree! I would love more feedback, particularly from anyone who has also been recently initiated into the field of AI.

> I hope that helps

Thank you Munashe!
